pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -209,7 +211,10 @@ func evictPodsFromTargetNodes(
 		}
 		klog.V(3).Infof("evicting pods from node %#v with usage: %#v", node.node.Name, node.usage)
 
-		nonRemovablePods, bestEffortPods, burstablePods, guaranteedPods := classifyPods(node.allPods, evictLocalStoragePods)
+		nonRemovablePods, bestEffortPods, burstablePods, guaranteedPods, err := classifyPods(ctx, client, node.allPods, evictLocalStoragePods)
+		if err != nil {
+			klog.Warningf("Unable to classify pods: %s", err)
+		}
```

Comment:
Log error and return? For all these pod lists are empty if error occurs, it makes no sense to do eviction.